src/util.cpp

```diff
@@ -78,6 +78,14 @@
 #include <openssl/conf.h>
 #include <thread>
 
+#ifdef WIN32
+#include <windows.h>    // for SetStdinEcho()
+#else
+#include <termios.h>    // for SetStdinEcho()
```

Comment:
is this generally supported on all UNIX, or does it need checks in the build system?

Reply:
termios.h? It seems to be pretty standard FWICT.